src/common/options/mon.yaml.in

```diff
@@ -172,7 +160,7 @@ options:
   flags:
   - runtime
   with_legacy: true
-- name: mon_cluster_log_file_level
+- name: mon_cluster_log_level
   type: str
   level: advanced
   desc: Lowest level to include is cluster log file
```

Comment:
I think "is" is a typo here but the description needs to be updated regardless to reflect that this option controls more than just the cluster log file now.